backend/src/main/java/codezap/template/repository/strategy/FullTextSearchSearchStrategy.java

```diff
@@ -32,6 +33,17 @@ public BooleanExpression matchedKeyword(String trimmedKeyword) {
         );
     }
 
+    private String parseKeyword(String trimmedKeyword) {
+        String[] parsedKeywords = trimmedKeyword.split(" ");
+        StringBuilder keywordBuilder = new StringBuilder();
+        for(String keyword : parsedKeywords) {
+            keywordBuilder.append("+");
+            keywordBuilder.append(keyword);
+            keywordBuilder.append(" ");
+        }
+        return keywordBuilder.toString();
```

Comment:
```suggestion
        return Arrays.stream(parsedKeywords)
                .map(keyword -> "+" + keyword)
                .collect(Collectors.joining(" "));
```

요렇게 개선해볼 수도 있겠네요 ㅎㅎ
상수 추출도 해볼 수 있을 것 같아요!